### PR TITLE
allow aws loadbalancers the option to select subnets.

### PR DESCRIPTION
**Pull request by alejandroEsc**
resolves: https://github.com/samsung-cnct/k2/issues/658

- Consistent spacing for the generated config file
- Code to allow subnets for load balancers, (search by name go get ids)
- Removed extraneous loadbalancer config.

### CODE REVIEW

**Comment by sostheim** [comment]
@mikeln commented in slack, build failed on 8min maximum run time limit.  Likely failing on AWS resource allocation slowness.  Should be okay to rerun as-is.

**Comment by sostheim** [comment]
retest this please

**Comment by alejandroEsc** [comment]
@davidewatson could you also review and merge if ok.

File: schemas/config/v1/definitions.json [hunk 1]
```diff
@@ -24,15 +24,6 @@
       "minItems": 1,
       "type": "array"
     },
-    "loadBalancerConfigs": {
```
Comment (coffeepac) [inline schemas/config/v1/definitions.json]: this is not extraneous.  please put it back

Comment (alejandroEsc) [inline schemas/config/v1/definitions.json]: Ahh I apologize, will do.

**Pull request merged**